src/ios/CDVSound.m

```diff
@@ -216,14 +216,14 @@ - (void)create:(CDVInvokedUrlCommand*)command
     NSString* mediaId = [command argumentAtIndex:0];
     NSString* resourcePath = [command argumentAtIndex:1];
 
-    CDVAudioFile* audioFile = [self audioFileForResource:resourcePath withId:mediaId doValidation:NO forRecording:NO];
+    CDVAudioFile* audioFile = [self audioFileForResource:resourcePath withId:mediaId doValidation:YES forRecording:NO];
```

Comment:
What's the reason we aren't doing validation anymore ? This leads to this code being skipped: https://github.com/gaelenh/cordova-plugin-media/blob/ios-no-audio-playback/src/ios/CDVSound.m#L160